php/libraries/NDB_Page.class.inc

```diff
@@ -776,12 +776,12 @@ class NDB_Page implements RequestHandlerInterface
                   $baseurl . '/js/polyfills.js',
                     // $baseurl . '/js/react/react-with-addons' . $min . '.js',
                     // $baseurl . '/js/react/react-dom' . $min . '.js',
-                  $baseurl . ($min === '') ?
+                  $baseurl . ($min === '' ?
                       '/vendor/js/react/react.development.js' :
-                      '/vendor/js/react/react.production.js',
-                  $baseurl . ($min === '') ?
+                      '/vendor/js/react/react.production' . $min . '.js'),
```

Comment:
why is this change done ?

Reply:
I think there was some warning about the order of operations in PHP. Like it evaluates the `.` before the `?` so it tries to concatenate a bool to a string which phan doesn't like.

If you look at the context of the change it basically just adds `$min` or an empty string. But if `$min` IS an empty string than you can just add its value cause it comes out to be the same thing.